shared/templates/template_ANSIBLE_service_disabled

```diff
@@ -4,19 +4,18 @@
 # complexity = low
 # disruption = low
 - name: Disable service {{{ SERVICENAME }}}
-  service:
-    name: "{{{ DAEMONNAME }}}"
+  systemd:
+    name: "{{{ DAEMONNAME }}}.{{ item }}"
     enabled: "no"
     state: "stopped"
+{{%- if MASK_SERVICE %}}
+    masked: "yes"
+{{%- endif %}}
   register: service_result
-  failed_when: "service_result is failed and ('Could not find the requested service' not in service_result.msg)"
+  failed_when: "service_result is failed and ('Could not find the requested service' not in service_result.msg) and ('Unit file {{{ DAEMONNAME }}}.{{ item }} does not exist' not in service_result.msg)"
```

Comment:
@matejak Unfortunately it fails on stopping when the socket file does not exist causing the ansible to fail. I tried to find some function in the module to check this condition, but couldn't find anything. Do you have any suggestions?